skins/cat17/faq-page/src/Faq.vue

```diff
@@ -71,167 +71,25 @@ export default {
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				},
-				{
-					name: 'Wikimedia-Projekte',
-					content: [
-						{
-							question: 'Wikimedia-Projekte Question',
-							answer: 'Wikimedia-Projekte Answer ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						},
-						{
-							question: 'Wikimedia-Projekte Question 2',
-							answer: 'Wikimedia-Projekte Answer 2 ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				},
-				{
-					name: 'Spendenverwendung',
-					content: [
-						{
-							question: 'Spendenverwendung Question',
-							answer: 'Spendenverwendung Answer ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						},
-						{
-							question: 'Spendenverwendung Question 2',
-							answer: 'Spendenverwendung Answer 2 ' +
-							'Lorem ipsum dolor sit amet, consectetur adipiscing elit, ' +
-							'sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. ' +
-							'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris ' +
-							'nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in ' +
-							'reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. '
-						}
-					]
-				}
-			]
+			topicTitle: '',
+			content: faqContent,
+			messages: faqMessages,
+			page: []
 		};
 	},
 	mounted: function () {
 		this.populatePageWithPreviewContent();
 	},
 	methods: {
 		populatePageByTopic: function ( topic ) {
-			this.page = topic;
+			this.page = this.content.questions.filter( ( question ) => question.topic.split().indexOf( topic.id ) !== -1 );
```

Comment:
I'm sorry but I disagree on this one :worried: 
The method is already clearly named and does only one thing -> populating the page with content by topic.
The anon function is not really `hasTopic`, as it is meant to filter by topic. Since `Array.filter` is already pretty clear what it does, I think this is readable enough. If we move this one line into a higher order function somewhere outside of the scope of the `Vue` element we're gonna create a small chaos IMHO. I really don't want to do that. If you insist on extracting this, I'd like it to be part of the `methods` object.

Reply:
Ok, it was more of a suggestion and readability is a subjective quality.

Don't forget to add the separator character, though.